app/src/main/java/org/simple/clinic/facility/change/confirm/ConfirmFacilityChangeEffectHandler.kt

```diff
@@ -31,6 +31,7 @@ class ConfirmFacilityChangeEffectHandler @AssistedInject constructor(
         .subtypeEffectHandler<ConfirmFacilityChangeEffect, ConfirmFacilityChangeEvent>()
         .addTransformer(ChangeFacilityEffect::class.java, changeFacility(schedulersProvider.io()))
         .addAction(CloseSheet::class.java, { uiActions.closeSheet() }, schedulersProvider.ui())
+        .addTransformer(LoadCurrentFacility::class.java, loadCurrentFacility())
```

Comment:
Why aren't we using the current facility dagger dependency for this, Is it because it won't be updated when the facility is changed or is there a different reason? 🤔

Reply:
We already have the facility repository which is being used to update the current facility, so I just decided to reuse that dependency instead of adding another one.